ui/job-view/headerbars/ReposMenu.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { getAllUrlParams } from '../../helpers/location';
+import { uiJobsUrlBase } from '../../helpers/url';
+
+export default class ReposMenu extends React.Component {
+  getRepoUrl(newRepoName) {
```

Comment:
Would this be best in `helpers/url.js` or here?

Reply:
Ahh, good point.  I'll move it.